ocaml/typing/typemexp.mli

```diff
@@ -0,0 +1,9 @@
+(** Interpret mode syntax as mode annotation, where axes can be left unspecified *)
```

Comment:
Good point - I don't like this either. The `typemexp.mli` to modes is what `typetexp.mli` to types. The latter contains `transl_*` which we follow.

Reply:
`typetexp.ml` may have "transl_" because "type" has a very salient different meaning in that module. Can you imagine `type_type_scheme` instead of `transl_type_scheme`? That reason wouldn't apply here.

"type_" is used more frequently than "transl_" as a prefix for a function that goes from the parsetree to the typedtree. I think that applies here too.

That said, I don't feel very strongly about this, so please decide one way or another and mark this as resolved.

Reply:
I'm fine with `transl`, actually, after looking at more modules.